samples/snippets/user_credentials.py

```diff
@@ -23,26 +23,16 @@
 import argparse
 
 
-def main(project, launch_browser=True):
+def main(project):
     # [START bigquery_auth_user_flow]
     from google_auth_oauthlib import flow
 
-    # TODO: Uncomment the line below to set the `launch_browser` variable.
-    # launch_browser = True
-    #
-    # The `launch_browser` boolean variable indicates if a local server is used
-    # as the callback URL in the auth flow. A value of `True` is recommended,
-    # but a local server does not work if accessing the application remotely,
-    # such as over SSH or from a remote Jupyter notebook.
-
+    # A local server is used as the callback URL in the auth flow.
     appflow = flow.InstalledAppFlow.from_client_secrets_file(
         "client_secrets.json", scopes=["https://www.googleapis.com/auth/bigquery"]
     )
 
-    if launch_browser:
-        appflow.run_local_server()
-    else:
-        appflow.run_console()
+    appflow.run_local_server()
```

Comment:
Let's add a comment pointing folks to gcloud and/or federated identity if they are working on a remote notebook or SSH.

```suggestion
    # This launches a local server to be used as the callback URL in the desktop
    # app auth flow. If you are accessing the application remotely, such as over
    # SSH or a remote Jupyter notebook, this flow will not work. Use the
    # `gcloud auth application-default login --no-browser` command or workload
    # identity federation to get authentication tokens, instead.
    # 
    appflow.run_local_server()
```